src/locales/ca_ES/achv.ts

```diff
@@ -271,6 +271,10 @@ export const PGMachv: AchievementTranslationEntries = {
   "FRESH_START": {
     name: "First Try!",
     description: "Complete the Fresh Start challenge."
+  },
+  "INVERSE_BATTLE": {
+    name: "Mirror rorriM",
```

Comment:
```suggestion
    name: "Mirror ɹoɹɹᴉW",
```

Idk if the font can render that :D

Reply:
do you want to change english translation?
( there is no ca_ES translation anymore? )

Reply:
Yeah you can replace the English translations of these. Make sure it shows up properly in game, there might be font issues (as mentioned). If it breaks then just leave it as-is.

Reply:
i will try it!

Reply:
sadly, we should update font later

<img width="1714" alt="image" src="https://github.com/user-attachments/assets/00faacae-c777-4115-bbbe-c4cb65a45101">

Reply:
This looks good enough for me 😂

Reply:
art team will kill me 🔫 . plz stop

btw, i updated others too! can you review it again? 🙇